mits/templates/mits_applications/game.html

```diff
@@ -0,0 +1,110 @@
+{% extends "mits_base.html" %}
+{% block body %}
+
+<h2>Enter Your Game Info</h2>
+
+<form method="post" action="game" class="form-horizontal" role="form">
+    {{ csrf_token() }}
+    <input type="hidden" name="id" value="{{ game.db_id }}" />
+
+    <div class="form-group">
+        <label class="col-sm-3 control-label">Name</label>
+        <div class="col-sm-6">
+            <input class="form-control focus" type="text" name="name" value="{{ game.name }}" />
+        </div>
+    </div>
+
+    <div class="form-group">
+        <label class="col-sm-3 control-label">Promo Blurb</label>
+        <div class="col-sm-6">
+            <input class="form-control focus" type="text" name="promo_blurb" value="{{ game.promo_blurb }}" />
+        </div>
+    </div>
+
+    <div class="form-group">
+        <label class="col-sm-3 control-label">Your Game's Genre:</label>
```

Comment:
This one label uses a colon ":" as a separator, as opposed to all the other labels.

Reply:
Good eye!  I've just fixed this.